episodes/3-documentation.md

```diff
@@ -346,6 +346,107 @@ MkDocs and Sphinx:
 - create nicely-formatted HTML pages out of .md or .rST files
 - programming language independent
 
+#### MkDocs
+
+::: challenge
+### Exercise: Generate MkDocs documentation
+
+Before we start generating documentation, make sure that you have installed mkdocs:
+
+```bash
+conda list mkdocs
+```
+
+Now, Lets create a new MkDocs project in the current project directory which contains the temperature conversion code: 
+
+```bash
+# In temperature-conversion
+mkdocs new .    
+```
+```output
+INFO    -  Writing config file: ./mkdocs.yml
+INFO    -  Writing initial docs: ./docs/index.md
+```
+
+This command creates a new MkDocs project in the current directory with a `docs` folder containing an `index.md` file 
+and a `mkdocs.yml` configuration file in the root of our project. If you look in the `index.md`, you can see that it currently contains default information about MkDocs.
+You can change this by writing a brief overview of the project or copying the contents of your `README.md`
+
+Now, let's fill in the `mkdocs.yml` configuration file for our project.
+
+```yaml
+site_name: Temperature Conversion
+
+theme:
+  name: "material"
+font: false
+
+nav:
+  - Overview: index.md
+
+plugins:
+  - search
+  - mkdocstrings
```

Comment:
Needed `pip install 'mkdocstrings[python]'` to enable build (and have it working with the API further down)